PFR/Main.lean

```diff
@@ -174,7 +174,8 @@ lemma PFR_conjecture_aux (h₀A : A.Nonempty) (hA : Nat.card (A + A) ≤ K * Nat
     PFR_conjecture_pos_aux h₀A hA
   let A' := A.toFinite.toFinset
   have h₀A' : Finset.Nonempty A' := by
-    simp [A', Finset.Nonempty]
+    simp only [Finset.Nonempty, Finite.toFinset_setOf, Finset.mem_filter, Finset.mem_univ, true_and,
+      A']
     exact h₀A
```

Comment:
```suggestion
    simp [A', Finset.Nonempty] using h₀A
```
is more robust to lemma renames

Reply:
I didn't think about that. Thank you very much! I'll take robustness to renaming in the future! 

I'll commit your suggestion later today when I'll be back home.

Reply:
Do you mean `simpa [A', Finset.Nonempty] using h₀A`, right? 

Done.

Reply:
Yes, sorry! Hard to review from a phone 😅